airflow/providers/microsoft/azure/operators/asb_queue.py

```diff
@@ -0,0 +1,206 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@josh-fell there is a draft PR for Azure service bus subscription operators (ASBCreateSubscriptionOperator, ASBUpdateSubscriptionOperator, ASBReceiveSubscriptionMessageOperator, ASBDeleteSubscriptionOperator) which uses the same `hooks/asb.py` which I thought I can keep it under `asb_subscription.py`  file

Reply:
If it make sense to group all operators under asb.py I'm OK with it but we don't have to.
My thumb rule for this is - it should be easily discoverable for users and easy to use.

In AWS for example, we grouped the S3 operators together because it's likely that in a single DAG you will need more than 1 operator.


So the question here: how distinct are queue operators from subscriptions and how likely they are to be used in a single DAG? That is a question only people who use it can answer. Either way - we can always change it with some community feedback.

Reply:
The message-sending functionality of a queue maps directly to a topic and its message-receiving functionality maps to a subscription, so there is a chance of user using the queue and subscription operators in the same DAG
 
https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions